_documentation/launch-on-startup.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Launch on Startup
+published: true
+---
+You can launch FocusCuts automatically at startup through System Preferences. In the Users & Groups, go to Login 
```

Comment:
Either remove `the` or add `tab` in the "In the Users & Groups" clause

Reply:
Users & Groups is a pane. Edited!